card/mod/machines/set/abstract/machine/output_update.rb

```diff
@@ -1,8 +1,10 @@
 def reset_machine_output
   Auth.as_bot do
     moc = machine_output_card
+    @updated_at = output_updated_at
     moc.delete! if moc.real?
     update_input_card
+    expire_if_source_file_changed @updated_at
```

Comment:
after the card is delete we can't get the updated_at date anymore. So we have to hold onto it here for